daml-lf/archive/da/daml_lf_1.proto

```diff
@@ -1139,5 +1142,6 @@ message Module {
 
 message Package {
   repeated Module modules = 1;
-  repeated string interned_package_ids = 2;
+  repeated string interned_package_ids = 2; /* Available since version 1.dev */
+  repeated DottedName interned_module_names = 3; /* Available since version 1.dev */
```

Comment:
Do we really need this field? All modules are listed in some specific order in the module they are defined in. Couldn't we just use the index in that list as the interned id?

Reply:
...wow, that's a good idea. I'll experiment, but as long as it doesn't have any self-package-reference problems, I'll use that instead.